test/sql/test_compiler.py

```diff
@@ -1396,12 +1397,24 @@ def test_nested_conjunctions_short_circuit(self):
                 and_(
                     or_(
                         or_(t.c.x == 12),
-                        and_(or_(), or_(and_(t.c.x == 8)), and_()),
+                        and_(or_(and_(t.c.x == 8))),
                     )
                 )
             ),
             "SELECT t.x FROM t WHERE t.x = :x_1 OR t.x = :x_2",
         )
+        with expect_deprecated():
```

Comment:
**mike bayer** (zzzeek) wrote:

all deprecation tests need to be in a test_deprecated.py suite so that they can be easily removed, so don't put expect_deprecated() stuff here.